### ISSUE TITLE
Teacher-forcing performance in test unseen

### ISSUE THREAD

**Issue by khanhptnk** [peteanderson80/Matterport3DSimulator#23]
Why didn't the paper report teacher-forcing performance in test unseen? (table 1).

**Comment by peteanderson80** [comment]
We only wanted to run one model on the test set. We used the validation set to determine the best training scheme (out of the options we tried), so we didn't need the test set to also confirm this. Over time, performing too many runs on the test set leaks information and leads to overfitting.
